collect_coins.md

```diff
@@ -0,0 +1,84 @@
+## WIP [CIP-25](https://gluwa.atlassian.net/browse/CSUB-25) CollectCoins
```

Comment:
CIP-25 -> CSUB-25, no ?

Reply:
It is a wordplay after bitcoin's improvement proposals. I will delete the file anyway after you have had a chance to review the general collect coins design.